doc/guide/accounting-guide.rst

```diff
@@ -368,7 +368,12 @@ urgency
 
 Thus the priority :math:`P` is calculated as follows:
 
-:math:`P = (F_{fairshare} \times W_{fairshare}) + (F_{queue} \times W_{queue}) + (F_{bank} \times W_{bank}) + (F_{urgency} - 16)`
+.. math::
+
+  P = (F_{fairshare} \times W_{fairshare})
+      + (F_{queue} \times W_{queue})
+      + (F_{bank} \times W_{bank})
+      + (W_{urgency} \times (F_{urgency} - 16))
```

Comment:
This might be slightly nicer visually if the factor comes before the weight like the other expressions.  Totally OK as is though!